images.go

```diff
@@ -0,0 +1,105 @@
+/*
+   Copyright (C) nerdctl authors.
+   Copyright (C) containerd authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/pkg/progress"
+	"github.com/containerd/containerd/platforms"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+)
+
+var imagesCommand = &cli.Command{
+	Name:   "images",
+	Usage:  "List images",
+	Action: imagesAction,
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:    "quiet",
+			Aliases: []string{"q"},
+			Usage:   "Only show numeric IDs",
+		},
+	},
+}
+
+func imagesAction(clicontext *cli.Context) error {
+	client, ctx, cancel, err := newClient(clicontext)
+	if err != nil {
+		return err
+	}
+	defer cancel()
+
+	var (
+		imageStore = client.ImageService()
+		cs         = client.ContentStore()
+	)
+
+	// To-do: Add support for --filter.
+	// Set filters to "" for now.
+	imageList, err := imageStore.List(ctx, "")
+	if err != nil {
+		return err
+	}
+
+	return printImages(ctx, clicontext, imageList, cs)
+}
+
+func printImages(ctx context.Context, clicontext *cli.Context, imageList []images.Image, cs content.Store) error {
+	quiet := clicontext.Bool("quiet")
+
+	w := tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
```

Comment:
Nit: stdout should be clicontext.App.Writer

Reply:
Does [`ps tabwriter`](https://github.com/AkihiroSuda/nerdctl/blob/master/ps.go#L70) needs to be fixed too?

Reply:
Fixed.